chain-signatures/node/src/rpc.rs

```diff
@@ -499,6 +518,7 @@ async fn try_publish_eth(
             tracing::info!(
                 request_id = ?CryptoHash(*request_id),
                 tx_hash = ?tx_hash,
+                elapsed = ?timestamp.elapsed(),
```

Comment:
Don't we want latency for Ethereum?

Reply:
that's in a TODO. I'm leaving it up to @ppca when she gets back